browser/sources.gni

```diff
@@ -493,6 +493,7 @@ brave_chrome_browser_sources += brave_browser_themes_sources
 brave_chrome_browser_sources += brave_browser_tor_sources
 brave_chrome_browser_sources += brave_browser_url_sanitizer_sources
 brave_chrome_browser_sources += brave_browser_web_discovery_sources
+brave_chrome_browser_sources += brave_browser_profiles_sources
```

Comment:
I'm not sure but could it be better to add sources and deps to `//chrome/browser/profiles` target?